dds/DCPS/ThreadMonitor.h

```diff
@@ -0,0 +1,79 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#ifndef OPENDDS_DCPS_THREAD_MONITOR_H
+#define OPENDDS_DCPS_THREAD_MONITOR_H
+
+#include "dcps_export.h"
+
+#if !defined (ACE_LACKS_PRAGMA_ONCE)
+#pragma once
+#endif /* ACE_LACKS_PRAGMA_ONCE */
+
+OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
+
+namespace OpenDDS {
+namespace DCPS {
+
+/**
+ * @class Thread_Monitor
+ *
+ * @brief Defines the means of tracking thread utilization by measuring
+ * time spent in event handling vs idle
+ */
+  class OpenDDS_Dcps_Export Thread_Monitor
```

Comment:
```suggestion
class OpenDDS_Dcps_Export ThreadMonitor
```

Things in the "main" namespace shouldn't have an indent and class names should be camel case.